spec/swagger_helper.rb

```diff
@@ -66,6 +66,9 @@
           SortingOptions: SORTING_OPTIONS,
         )
       end
+      if Rails.env.in?(%w[review])
```

Comment:
Not sure this worked 🤔

Reply:
Hi @cwrw It didn't work.. the idea was to add the review app url into the list of servers. Just removed. tks